src/csharp/Grpc.Tools.CSharpGenerator/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Grpc.Tools.CSharpGenerator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                var projectDir = args[0];
+                var intermediateOutputDir = args[1];
+
+                var osName = string.Empty;
+
+                if (Grpc.Core.Internal.PlatformApis.IsWindows)
+                    osName = "windows_{0}";
+                else if (Grpc.Core.Internal.PlatformApis.IsMacOSX)
+                    osName = "macosx_{0}";
+                else if (Grpc.Core.Internal.PlatformApis.IsLinux)
+                    osName = "linux_{0}";
+                else
+                {
+                    Console.Error.WriteLine("Could not detect your operating system. The only supported operating systems are Windows, Mac OSX and Linux");
+                    return;
+                }
+
+                var bitness = string.Empty;
+                if (Grpc.Core.Internal.PlatformApis.Is64Bit)

```

Comment:
Just wondering why is it important to worry about the bitness. Most tools are 32 bit (the C# compiler and Visual Studio are 32 bit programs, as are good 90% of Windows binaries), and do not generally need to match the bitness of binaries they generate. Are `protoc` and `grpc_csharp_plugin` binaries need to match the bitness of *target* code they generate? @jtattermusch, what are your thoughts?